cmd/database.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"database/sql"
+	"fmt"
+	"xkcdcomics/model"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+var Db *sql.DB
+
+func SetupDatabase() (*sql.DB, error) {
+	Db, err := sql.Open("sqlite3", "./xkcd_comics.db")
+	if err != nil {
+		return nil, fmt.Errorf("failed to open database: %v", err)
+	}
+
+	// Create table if not exists
+	_, err = Db.Exec(`CREATE TABLE IF NOT EXISTS comics (
+		num INTEGER PRIMARY KEY,
+		safeTitle TEXT,
+		img TEXT,
+		title TEXT,
+		transcript TEXT
+	)`)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create table: %v", err)
+	}
+
+	return Db, nil
```

Comment:
As we don't expose the db connection, just return the error and then the rest of the functions can just use the global, unexported ```db``` object